content/logo/index.html.haml

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Jenkins Logos
```

Comment:
It may make the impression that all this images are official logos of the Jenkins project. They are not.

I suggest renaming this page to Artwork and then put it under the "Resources" tab on the top panel. Such generic name would allow to use non-logo images as well though they should rather be grouped below logos (new `h1`).